proposals/fluentd.adoc

```diff
@@ -0,0 +1,112 @@
+== Fluentd Proposal
+
+*Name of project:* Fluentd
+
+*Description:* Fluentd is an open source data collector for unified logging layer. Fluentd's 600+ plugins connect it to many data sources and data outputs. Fluentd has a large adopter community consisting of Atlassian, LINE, Microsoft, Nintendo, GREE (and more).
```

Comment:
It would be nice to talk about community beyond just adopters. Do these companies contribute back?